web/views/bundles_view.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Cog.V1.BundlesView do
+  use Cog.Web, :view
+
+  alias Ecto.Association
+
+  def render("bundle.json", %{bundle: bundle}) do
+    %{id: bundle.id,
+      name: bundle.name,
+      namespace: bundle.namespace.name,
+      enabled: bundle.enabled,
+      relay_groups: render_relay_groups(bundle.relay_groups),
+      inserted_at: bundle.inserted_at,
+      updated_at: bundle.updated_at}
+  end
```

Comment:
`commands` should also be listed here. The `show` endpoint expects commands from the detailed query.